src/Avalonia.Base/Input/TouchDevice.cs

```diff
@@ -59,16 +60,18 @@ public void ProcessRawEvent(RawInputEventArgs ev)
                 else
                 {
                     var settings = AvaloniaLocator.Current.GetRequiredService<IPlatformSettings>();
+                    var doubleClickTime = settings?.GetDoubleTapTime(PointerType.Touch).TotalMilliseconds ?? 500;
```

Comment:
How it can be null, if GetRequiredService would throw an exception?

Reply:
Yeah I copy/pasted this code from `MouseDevice` which uses `GetService`. We should just use `GetRequiredService` throughout.